src/coupling.c

```diff
@@ -21,70 +23,166 @@
 #include <math.h>
 #include "headers.h"
 
-//-----------------------------------------------------------------------------
-//  Local functions
-//-----------------------------------------------------------------------------
-static void   coupling_findNodeInflow(int j, double tStep);
-static void   coupling_adjustInflows(int j, double inflowAdjustingFactor);
-static void   opening_findCouplingType(TCoverOpening* opening, double crestElev,
-                                       double nodeHead, double overlandHead);
-static void   opening_findCouplingInflow(TCoverOpening* opening, double crestElev,
-                                         double nodeHead, double overlandHead);
-
```

Comment:
I understand those declarations are moved to their own header file, is that right?

Reply:
Yes, exactly. They are moved to a separate header file where they are declared as extern C (for C-language calling convention) when used from C++ code.